Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1695,17 +1704,99 @@ def get_endpoints_command(client, args):
     )
 
 
-def return_endpoint_standard_context(endpoints):
-    endpoints_context_list = []
-    for endpoint in endpoints:
-        endpoints_context_list.append(assign_params(**{
-            "Hostname": (endpoint['host_name'] if endpoint.get('host_name', '') else endpoint.get('endpoint_name')),
-            "ID": endpoint.get('endpoint_id'),
-            "IPAddress": endpoint.get('ip'),
-            "Domain": endpoint.get('domain'),
-            "OS": endpoint.get('os_type'),
-        }))
-    return endpoints_context_list
+def get_trasnformed_dict(old_dict, transformation_dict):
+    """
+        Returns a dictionary with the same values as old_dict, with the correlating key:value in transformation_dict
+
+        :type old_dict: ``dict``
+        :param old_dict: Old dictionary to pull values from
+
+        :type transformation_dict: ``dict``
+        :param transformation_dict: Transformation dictionary that contains oldkeys:newkeys
+
+        :return Transformed dictionart (according to transformation_dict values)
+        :rtype ``dict``
+    """
+    new_dict = {}
+    for k in list(old_dict.keys()):
+        if k in transformation_dict:
+            new_dict[transformation_dict[k]] = old_dict[k]
+    return new_dict
+
+
+def convert_os_to_standart(endpoint_os):
+    os_type = ''
+    if 'windows' in endpoint_os.lower():
+        os_type = "Windows"
+    elif 'linux' in endpoint_os.lower():
+        os_type = "Linux"
+    elif 'macos' in endpoint_os.lower():
+        os_type = "Macos"
+    elif 'android' in endpoint_os.lower():
+        os_type = "Android"
+    return os_type
```

Comment:
shouldn't this be in CSP as a static func of the endpoint class?

Reply:
don't think so, relevant currently to xdr only